topology/probes/k8s/graph.go

```diff
@@ -41,6 +41,10 @@ const (
 	nodeNameField = detailsField + ".Spec.NodeName"
 )
 
+func lookupChildren(g *graph.Graph, parent, childFilter *graph.Node, edgeFilter graph.Metadata) []*graph.Node {
```

Comment:
what's the sens of this "C binding" function ?

Reply:
removed function (was not used in this PR context)